test/libyul/objectCompiler/wasm/no_main_function.yul

```diff
@@ -19,4 +19,4 @@
 // )
 //
 // Binary:
-// 0061736d01000000010401600000020100030201000503010001060100070a01066d656d6f727902000a0801060002401a0b0b
```

Comment:
Why did it get longer?

Reply:
https://github.com/ethereum/solidity/pull/10454#discussion_r533383239

It was not including the arguments to drop. Which in the above cases were side-effect free, but nothing ensured that.